docs/resources.md

```diff
@@ -0,0 +1,53 @@
+#### Arguments for AKSClusterConfigSpec is the spec for a AKSClusterConfig resource
+
+The following arguments are supported:
+
+* `Imported` - (Optional) Is AKS cluster imported? Defaul: `false` (bool)
```

Comment:
These descriptions can be moved to comments on the `AKSClusterConfigSpec` and with the additional of the kubebuilder validation annotations.